doc/fastly-configuration.rst

```diff
@@ -13,7 +13,7 @@ Tagging
 ~~~~~~~
 
 Fastly supports cache tagging out of the box.
-Configure the tag header to ``Surrogate-Key``. (``fos_http_cache.tags.response_header`` if you use FOSHttpCacheBundle)
+Configure the tag header to ``Surrogate-Key``. (``fos_http_cache.tags.response_header`` as well as ``fos_http_cache.tags.separator`` to `` `` if you use FOSHttpCacheBundle)
```

Comment:
```suggestion
Configure the tag header to ``Surrogate-Key`` and the separator to `` `` (empty string).

If you use FOSHttpCacheBundle, this can be done in the bundle configuration:

.. code-block:: yaml

    fos_http_cache:
        tags:
            response_header: Surrogate-Key
            separator: ' '

If you do not use the bundle, you need to pass your own instance of the TagHeaderFormatter with the right options to the ResponseTagger::

    use FOS\HttpCache\ResponseTagger;
    use FOS\HttpCache\TagHeaderFormatter\CommaSeparatedTagHeaderFormatter;
    
    // ...
    
    $tagger = new ResponseTagger([
        'header_formatter' => new CommaSeparatedTagHeaderFormatter('Surrogate-Key', ' '),
    ]);
```